service/storage_proxy.cc

```diff
@@ -170,7 +170,7 @@ class storage_proxy::remote {
 
         ser::storage_proxy_rpc_verbs::register_counter_mutation(&_ms, std::bind_front(&remote::handle_counter_mutation, this));
         ser::storage_proxy_rpc_verbs::register_mutation(&_ms, std::bind_front(&remote::receive_mutation_handler, this, sp->_write_smp_service_group));
-        ser::storage_proxy_rpc_verbs::register_hint_mutation(&_ms, [this, sp] <typename... Args>(Args&&... args) { return receive_mutation_handler(sp->_hints_write_smp_service_group, std::forward<Args>(args)..., std::monostate()); });
+        ser::storage_proxy_rpc_verbs::register_hint_mutation(&_ms, [this, sp] <typename... Args>(Args&&... args) { return receive_mutation_handler(sp->_hints_write_smp_service_group, std::forward<Args>(args)..., std::monostate(), rpc::optional<fencing_token>{}); });
```

Comment:
FIXME?

Reply:
There are todo's in my personal todo list to fix this and other RPCs, they will be done right after this PR, fixme's in code won't help much.

Reply:
personal todo lists have a small bus factor

Reply:
I can create a ticket with bullets of what needs to be done.

Reply:
https://github.com/scylladb/scylladb/issues/14070